### ISSUE TITLE
Is it possible to do this on a retail ps4?

### ISSUE THREAD

**Issue by laura240406** [idc/ps4-pup_decrypt#5]

**Comment by laura240406** [comment]
Or is it possible to do it on a PC somehow?

**Comment by deadmeme5** [comment]
> Or is it possible to do it on a PC somehow?

yes you can use the orbital emulator im working on right now to emulate. It works mostly good with a rooted/jailbroken ps4 on which you can add custom software. Though, on orbital emulator is kinda slow rn.
